wordpress-mobile/WordPress-Android
# Issue/update format icons

### Fix
1. Add horizontal rule format button to the editor format toolbar as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6045.

2. Update editor format toolbar button icons including:
- [`gridicons-code`](https://github.com/Automattic/gridicons/blob/master/sources/svg/gridicons-code.svg)
- [`gridicons-next-page`](https://github.com/Automattic/gridicons/blob/master/sources/svg/gridicons-next-page.svg)
- [`gridicons-read-more`](https://github.com/Automattic/gridicons/blob/master/sources/svg/gridicons-read-more.svg)
- [`gridicons-underline`](https://github.com/Automattic/gridicons/blob/master/sources/svg/gridicons-underline.svg)

3. Update editor format toolbar button state color to:

State | Name | Value
-|-|-
Active | `#2e4453` | [`grey_dark`](https://wordpress.com/design-handbook/colors/#colors-grays)
Inactive | `#87a6bc` | [`grey`](https://wordpress.com/design-handbook/colors/#colors-grays)
Disabled | `#c8d7e1` | [`grey_lighten_20`](https://wordpress.com/design-handbook/colors/#colors-grays)

### Test
0. Enable ***Beta*** editor.
1. Go to ***Sites*** tab.
2. Tap ***Create*** floating button.
3. Tap content text field.
4. Scroll through format toolbar horizontally.
5. Notice updated format toolbar button icons.

#### Note
The horizontal row button will not be shown until https://github.com/wordpress-mobile/AztecEditor-Android/pull/384 is merged and the version containing those changes is included in the [WordPressEditor dependencies](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/build.gradle#L54).
